packages/codemirror/src/editor.ts

```diff
@@ -1368,6 +1381,30 @@ namespace Private {
       case 'lineHeight':
         el.style.lineHeight = value ? value.toString() : null;
         break;
+      case 'lineNumbers':
+        let currentGutters = editor.getOption('gutters').slice() as Array<
+          string
+        >;
+        const lineNoIndex = currentGutters.indexOf('CodeMirror-linenumbers');
+        if (lineNoIndex < 0 && value) {
+          currentGutters.unshift('CodeMirror-linenumbers');
+        } else if (lineNoIndex === 0 && !value) {
+          currentGutters.shift();
```

Comment:
Are there any other gutters that people might want to add? This logic could get difficult to read if there are more than two possible gutters. I am not sure if there is a better way, however, so happy to leave it as is...